src/parse_eqs.jl

```diff
@@ -340,9 +340,17 @@ function _newfnbody(fnbody, fnargs, d_indx)
                 append!(v_newindx, tmp_newindx)
                 append!(v_arraydecl, tmp_arraydecl)
             elseif ex_head == :if
+                # The first argument of an `if` expression is the condition, the
+                # second one is the block the condition is true, and the third
+                # is the `else`-block or an `ifelse`-block.
                 push!(newfnbody.args, Expr(:if, ex.args[1]))
                 for exx in ex.args[2:end]
-                    ifbody, tmp_newindx, tmp_arraydecl = _newfnbody( exx, fnargs, d_indx)
+                    if exx.head == :elseif
+                        exxx = Expr(:block, Expr(:if, exx.args[1].args[2], exx.args[2:end]...))
```

Comment:
Nice workaround! I still can't figure out why `:elseif` blocks can't be treated straight-forward as `:if` blocks 🤷‍♂